full-service/src/service/wallet_service.rs

```diff
@@ -83,4 +95,11 @@ impl<
             logger,
         }
     }
+
+    pub fn get_conn(&self) -> Result<Conn, WalletDbError> {
```

Comment:
Nice - I see, so for any service that asks for a wallet DB connection, it will return `WalletFunctionsDisabled`, but anything accessing just the ledger will work fine. Elegant 👏

Reply:
Yeah, that seemed _so_ much better than figuring it out on a per call basis